.github/workflows/build-lint-test.yml

```diff
@@ -11,8 +11,17 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-
+    env:
+      PGHOST: localhost
+      PGDATABASE: preflight_test_project_next_js_passing
+      PGUSERNAME: preflight_test_project_next_js_passing
+      PGPASSWORD: preflight_test_project_next_js_passing
     steps:
+      - uses: ikalnytskyi/action-setup-postgres@v4
+        with:
+          username: ${{ env.PGUSERNAME }}
+          password: ${{ env.PGPASSWORD }}
+          database: ${{ env.PGDATABASE }}
```

Comment:
set up database for usage in tests (Preflight runs ESLint in [the `preflight-test-project-next-js-passing` project](https://github.com/upleveled/preflight-test-project-next-js-passing), which includes SafeQL because of the `postgres` dependency, which requires database credentials)